library/Versions.re

```diff
@@ -145,3 +146,20 @@ let getRemoteVersions = () => {
      )
   |> Lwt.return;
 };
+
+let throwIfInstalled = versionName => {
+  switch(getInstalledVersions()) {
+  | Ok(x) => {
+    Array.to_list(x)
+    |> List.exists(x => {
+      open Local;
```

Comment:
opening `Local` here feels weird: maybe we can move this function to the `Local` module? I mean, _it is_ something related to local versions only

`Versions.Local.throwIfInstalled` sounds reasonable to me - and no `open` will be needed!

Reply:
I agree it! and I try it... but `getInstalledVersions()` belong to `Version` module, so I cannot use it in `Versions.Local`. I Tried to write  `Version.getInstalledVersions()` but it caused build error.
How can I use parent module's function in child? (If this resolution is in [docs](https://reasonml.github.io/docs/en/module), I'm so sorry)

Reply:
You're totally right. I think it's not something related to parent-child, but more of the order of declaration: https://reasonml.github.io/try?rrjsx=true&reason=DYUwLgBA7g9gTsAJhAvBARLBj0G4BQ+AtjIgK6gQASIwwMqEA3vhBKJAA6MAUAlKgB8ETnACWAOzAB9EBMTBJIHuhp0G6CAGot0eEj4EAvoTX0AdJ364gA

I'd say we leave it as it is right now, and later on we'll have to refactor. In my opinion, `Versions.getInstalledVersions` should be `Versions.Local.getInstalled` (makes more sense), but it shouldn't bother you now